CHANGELOG.rst

```diff
@@ -1,5 +1,56 @@
 .. _changelog:
 
+0.51.0
+------
+
+Renku ``0.51.0`` introduces new features related to Renku 2.0. However, Renku 2.0 is still
+in early development and is not yet accessible to users. For more information, see our
+[roadmap](https://github.com/SwissDataScienceCenter/renku-design-docs/blob/main/roadmap.md).
+
+1. This release introduces *groups* to Renku 2.0.
+2. Various bug fixes and improvements
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- **UI**: A new entity, *groups*, has been introduced to Renku 2.0. Groups are a way to organize
+  projects in Renku 2.0.
+- **UI**: Projects are always in a group -- either the user's implicitly-created group, or a group
```

Comment:
I don't want to nitpick here and this is fine for now but we really need some clarity on what we're calling these things. We've gone from everything having a "namespace" to now everything being a "group" and a user belonging to an implicitly-created single-user group. I think this is unnecessarily complicated and needs to be revised asap in the upcoming builds/releases.

Reply:
Oops I missed that, I shouldn't have let that through